src/components/tables/table-children/TableChildBlankSlate.tsx

```diff
@@ -28,7 +28,8 @@ export const TableChildBlankSlate = (props: ITableChildBlankSlateProps): JSX.Ele
   if (tableCompositeState.isInError) {
     blankSlatePropsToUse = blankSlateOnError || blankSlateDefault;
   } else if (tableCompositeState.filter
-    || _.some(tableCompositeState.predicates, (value: any) => !_.isUndefined(value) && value !== TABLE_PREDICATE_DEFAULT_VALUE)) {
+    || _.some(tableCompositeState.predicates, (value: any) => !_.isUndefined(value) && value !== TABLE_PREDICATE_DEFAULT_VALUE)
+    || tableCompositeState.from || tableCompositeState.to) {
```

Comment:
I would do a carriage return before the second `||`

Reply:
Should I put `|| (tableCompositeState.from && tableCompositeState.to))`? since it's an both or none kind of situation

Reply:
I'd say we should leave it with the OR, because maybe in the future the datepicker could apply only one the limits.